frontend/src/components/ProjectMember/ProjectMemberRolePanel/RoleDescription.vue

```diff
@@ -51,7 +51,11 @@ const gotoIssuePage = async () => {
     return;
   }
 
-  window.open(`/issue/${issueSlug(issue.title, issue.uid)}`, "_blank");
+  const projectId = extractProjectResourceName(issue.name);
+  window.open(
```

Comment:
I'd like to use `router.resolve` with the route name to generate the URL, so that we can avoid hard code the path

Reply:
Done